frog/imports/api/activities.js

```diff
@@ -39,50 +50,50 @@ export const addActivity = (activityType, data = {}, id, groupingKey) => {
   }
 };
 
-export const duplicateActivity = activity =>
+export const duplicateActivity = (activity: ActivityDbT) =>
   Activities.insert({
     ...activity,
     _id: uuid(),
     data: { ...activity.data, name: activity.data.name + ' (copy)' }
   });
 
-export const addGraphActivity = params =>
+export const addGraphActivity = (params: Object) =>
   Activities.insert({
     ...params,
     graphId: params.graphId,
```

Comment:
I believe it is, I just fixed the type annotation without looking deeply at all the functions.

The slight difference could be that we still write a field `graphId: undefined`. Not sure how mongo behaves with this